src/main/java/datawave/accumulo/inmemory/InMemoryTableOperations.java

```diff
@@ -439,6 +440,10 @@ public void clone(String srcTableName, String newTableName, boolean flush, Map<S
         throw new NotImplementedException();
     }
     
+    @Override
+    public void clone(String s, String s1, CloneConfiguration cloneConfiguration)
+                    throws AccumuloException, AccumuloSecurityException, TableNotFoundException, TableExistsException {}
+    
```

Comment:
Would it be better to `throw new NotImplementedException();` here instead?

Reply:
Good point -- changed the code.